src/integrationTest/kotlin/com/sourcegraph/cody/util/CodyIntegrationTextFixture.kt

```diff
@@ -185,9 +185,10 @@ open class CodyIntegrationTextFixture : BasePlatformTestCase() {
     }
   }
 
-  protected fun activeSession(): FixupSession {
+  protected fun activeSession(): FixupSessionV1 {
     assertActiveSession()
-    return FixupService.getInstance(project).getActiveSession()!!
+    // TODO: This is a bit hacky while there's two versions floating around
```

Comment:
Is there any reason we need that casting there instead of just returning `FixupSession`?